src/components/content/forms/drawer-form.tsx

```diff
@@ -50,6 +51,29 @@ export const DrawerForm = <T,>({
     event.preventDefault();
     setIsSubmitting(true);
     const form = event.target;
+
+    const formDataValidation = (event.target as HTMLElement).querySelectorAll(
+      "input"
+    );
+    const formDataValidationToArray = Array.from(formDataValidation);
```

Comment:
```suggestion
    const inputs = Array.from((event.target as HTMLElement).querySelectorAll(
      "input"
    ));
```

`formDataValidation` and `formDataValidationToArray` are really awkward names. `formDataValidation` does not point to any type of validation, it points to inputs. `formDataValidationToArray` doesn't point to anything that will transform it, e.g. `to array` it. It instead points to an array. Generally it is good practice to not have the type included in the name variables as we can easily see the type. You also do all of this in one pass to avoid the extra variable, and just call it `inputs` or `inputElements` to make it clear these are DOM elements.